From 849573932008b9ce0b26226c01286c53b08ac8d3 Mon Sep 17 00:00:00 2001
From: Hiep Pham <hiep.pham.zy@renesas.com>
Date: Thu, 18 Mar 2021 08:43:21 +0700
Subject: [PATCH 122/135] arm64: dts: renesas: r9a07g044l: Correct compatible
 CANFD.

This patch to remove rcar compatible of canfd driver.

Signed-off-by: Hiep Pham <hiep.pham.zy@renesas.com>
---
 arch/arm64/boot/dts/renesas/r9a07g044l.dtsi | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi b/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi
index 60b0b98..a1598ab 100644
--- a/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi
+++ b/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi
@@ -296,8 +296,7 @@
 		};
 
 		can: can@10050000 {
-			compatible = "renesas,rcar-gen3-canfd",
-					"renesas,r9a07g044l-canfd";
+			compatible = "renesas,r9a07g044l-canfd";
 			reg = <0 0x10050000 0 0x6000>;
 			interrupts =    <GIC_SPI 426 IRQ_TYPE_LEVEL_HIGH>,
 					<GIC_SPI 422 IRQ_TYPE_LEVEL_HIGH>,
-- 
2.7.4

